flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/WindowAggregateITCase.scala

```diff
@@ -319,6 +319,48 @@ class WindowAggregateITCase(mode: StateBackendMode)
     assertEquals(expected.sorted, sink.getRetractResults.sorted)
   }
 
+  // used to verify compile works normally when constants exists in group window key (FLINK-17553)
+  @Test
+  def testWindowAggregateOnConstantValue(): Unit = {
+    val ddl1 =
+      """
+        |CREATE TABLE src (
+        |  log_ts STRING,
+        |  ts TIMESTAMP(3),
+        |  a INT,
+        |  b DOUBLE,
+        |  rowtime AS CAST(log_ts AS TIMESTAMP(3)),
+        |  WATERMARK FOR rowtime AS rowtime - INTERVAL '0.001' SECOND
+        |) WITH (
+        |  'connector' = 'COLLECTION',
+        |  'is-bounded' = 'false'
+        |)
+      """.stripMargin
+    val ddl2 =
+      """
+        |CREATE TABLE dst (
+        |  ts TIMESTAMP(3),
+        |  a BIGINT,
+        |  b DOUBLE
+        |) WITH (
+        |  'connector.type' = 'filesystem',
+        |  'connector.path' = '/tmp/1',
+        |  'format.type' = 'csv'
+        |)
+      """.stripMargin
+    val query =
+      """
+        |INSERT INTO dst
+        |SELECT TUMBLE_END(rowtime, INTERVAL '0.003' SECOND), COUNT(ts), SUM(b)
+        |FROM src
+        | GROUP BY 'a', TUMBLE(rowtime, INTERVAL '0.003' SECOND)
+      """.stripMargin
+    tEnv.sqlUpdate(ddl1)
+    tEnv.sqlUpdate(ddl2)
+    tEnv.sqlUpdate(query)
+    tEnv.explain(true)
```

Comment:
Agreed.  Thanks for you suggestion.
Here I just want to verify that explain work normally instead of checking the result, but I don't find  a proper test file to place it. Which file would you suggest me to place?

Reply:
I would suggest to verify the result. Because you already added a plan test. 
You can use the following to verify the result. 

```scala
val sink = new TestingAppendSink
tEnv.sqlQuery(sql).toAppendStream[Row].addSink(sink)
env.execute()
```

Reply:
I got your meaning.